classes/class-p4-loader.php

```diff
@@ -52,6 +52,89 @@ public static function get_instance( $services = [] ) : P4_Loader {
 	private function __construct( $services ) {
 		$this->load_files();
 		$this->load_services( $services );
+		add_filter( 'wp_kses_allowed_html', [ $this, 'filter_allowed_html' ], 10, 2 );
+	}
+
+	/**
+	 * Get allowed tags and add some more
+	 *
+	 * @param string $context context.
+	 * @return mixed
+	 */
+	public function filter_allowed_html( $context = '' ) {
+		global $allowedposttags;
+		$allowed_tags = $allowedposttags;
+		// Add our own tags.
+		$allowed_tags['figure'] = array();
+		$allowed_tags['style']  = array(
+			'background'            => true,
```

Comment:
The attributes here are not for the element's content, only for its attributes. So it should work with just an empty array.